src/test/java/com/flab/lemonticket/service/EventServiceTest.java

```diff
@@ -0,0 +1,115 @@
+import static org.mockito.Mockito.*;
+import static org.mockito.internal.verification.VerificationModeFactory.times;
+
+public class EventServiceTest {
+    @Mock
+    private EventService eventService;
+
+    @InjectMocks
+    private EventController eventController;
+
+    @BeforeEach
+    void setUp(){
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @Test
+    @DisplayName("addEvent 메소드가 공연 정보를 성공적으로 추가하는지 테스트")
+    void testAddEvent() throws IOException {
+        Event event = new Event();
+        event.setEventId(1L);
+        event.setTitle("Test Event");
+        event.setCreatedAt(now());
+        event.setUpdatedAt(now());
+
+        List<EventTime> eventTimes = Collections.singletonList(new EventTime());
+        List<SeatLayout> seatLayouts = Collections.singletonList(new SeatLayout());
+        List<PerformerEvent> performerEvents = Collections.singletonList(new PerformerEvent());
+        List<MultipartFile> eventImgs = Collections.emptyList();
+        MultipartFile profileImgFile = mock(MultipartFile.class);
+
+        when(eventService.addEvent(any(Event.class), anyList(), anyList(), anyList(), anyList(), any(MultipartFile.class)))
+                .thenReturn(Map.of("event", event));
+
+        Map<String, Object> response = eventService.addEvent(event, eventTimes, seatLayouts, performerEvents, eventImgs, profileImgFile);
+
+        assertNotNull(response);
+        assertEquals(event, response.get("event"));
+        verify(eventService, times(1)).addEvent(any(Event.class), anyList(), anyList(), anyList(), anyList(), any(MultipartFile.class));
+    }
+
+    @Test
+    @DisplayName("modifyEvent 메소드가 공연 정보를 성공적으로 수정하는지 테스트")
+    void testModifyEvent() throws IOException {
+        Event event = new Event();
+        event.setEventId(1L);
+        event.setTitle("Updated Event");
+        event.setCreatedAt(LocalDateTime.now());
+        event.setUpdatedAt(LocalDateTime.now());
+
+        List<EventTime> eventTimes = Collections.singletonList(new EventTime());
+        List<SeatLayout> seatLayouts = Collections.singletonList(new SeatLayout());
+        List<PerformerEvent> performerEvents = Collections.singletonList(new PerformerEvent());
+        List<MultipartFile> eventImgs = Collections.emptyList();
+        MultipartFile profileImgFile = mock(MultipartFile.class);
+
+        when(eventService.modifyEvent(any(Event.class), anyList(), anyList(), anyList(), anyList(), any(MultipartFile.class)))
+                .thenReturn(Map.of("event", event));
+
+        Map<String, Object> response = eventService.modifyEvent(event, eventTimes, seatLayouts, performerEvents, eventImgs, profileImgFile);
+
+        assertNotNull(response);
+        assertEquals(event, response.get("event"));
+        verify(eventService, times(1)).modifyEvent(any(Event.class), anyList(), anyList(), anyList(), anyList(), any(MultipartFile.class));
+    }
+
+    @Test
+    @DisplayName("searchEvents 메소드가 공연 정보를 성공적으로 검색하는지 테스트")
```

Comment:
```suggestion
    @DisplayName("공연 제목으로 검색을 했을때 결과 조회가 가능하다.")
```

given/when/then 의 내용이 함축된 하나의 문장 형태로 구성을 하는게 좋을것 같아요.